optionals/compat_optre/ACE_Medical_Actions.hpp

```diff
@@ -0,0 +1,78 @@
+class ACE_Head {
```

Comment:
You can use a macro for those long config classes.

Eg:
```
#define MACRO_BIOFOAM(var) \
    class Biofoam { \
        displayName = CSTRING(Biofoam); \
        distance = 2.0; \
        condition = QUOTE([ARR_4(_player, _target, var, 'Biofoam')] call DEFUNC(medical,canTreatCached)); \
        statement = QUOTE([ARR_4(_player, _target, var, 'Biofoam')] call DEFUNC(medical,treatment)); \
        exceptions[] = {}; \
        showDisabled = 1; \
        priority = 2; \
        hotkey = ""; \
        icon = PATHTOF(UI\icons\Biofoam.paa); \
    };

class ACE_Head {
    MACRO_BIOFOAM('head')
};
```

Or even with added body part:
```
#define MACRO_BIOFOAM(var1,var2) \
    class ACE_var1 { \
        class Biofoam { \
            displayName = CSTRING(Biofoam); \
            distance = 2.0; \
            condition = QUOTE([ARR_4(_player, _target, var2, 'Biofoam')] call DEFUNC(medical,canTreatCached)); \
            statement = QUOTE([ARR_4(_player, _target, var2, 'Biofoam')] call DEFUNC(medical,treatment)); \
            exceptions[] = {}; \
            showDisabled = 1; \
            priority = 2; \
            hotkey = ""; \
            icon = PATHTOF(UI\icons\Biofoam.paa); \
        }; \
    };

MACRO_BIOFOAM(Head,'head')
```

(Untested, could contain errors!)